.gitignore

```diff
@@ -1,3 +1,4 @@
 dist
 node_modules
-.sizecache.json
\ No newline at end of file
+.sizecache.json
+*.bak
```

Comment:
You should put this in your own `core.excludesfile` setting. Individual projects should not have rules for specific editors.